examples/titanic_example/plot_titanic.py

```diff
@@ -301,15 +299,14 @@
 train_data_sample_inputs = [InputRef(identifier="datasamples", asset_key=key) for key in train_data_sample_keys]
 test_data_sample_inputs = [InputRef(identifier="datasamples", asset_key=key) for key in test_data_sample_keys]
 
-traintuple = TraintupleSpec(
+traintuple = TaskSpec(
```

Comment:
We should remove the tuple. 
```suggestion
train_task = TaskSpec(
```